Sources/ThreadHelper.swift

```diff
@@ -26,8 +26,12 @@
 
 import Foundation
 
-func dispatch_async_safely_main_queue(block: ()->()) {
-    if NSThread.isMainThread() {
+func dispatch_async_safely(queue: dispatch_queue_t?, block: ()->()) {
+    if let queue = queue {
```

Comment:
If the main queue is passed in, the following `NSThread.isMainThread()` statement will never executed, which may cause a flickering.